nvm.sh

```diff
@@ -53,7 +53,7 @@ nvm_command_info() {
     INFO="$(which "${COMMAND}") ($(type "${COMMAND}" | command awk '{ $1=$2=$3=$4="" ;print }' | command sed -e 's/^\ *//g' -Ee "s/\`|'//g" ))"
   elif type "${COMMAND}" | nvm_grep -q "^${COMMAND} is an alias for"; then
     INFO="$(which "${COMMAND}") ($(type "${COMMAND}" | command awk '{ $1=$2=$3=$4=$5="" ;print }' | command sed 's/^\ *//g'))"
-  elif type "${COMMAND}" | nvm_grep -q "^${COMMAND} is \/"; then
+  elif type "${COMMAND}" | nvm_grep -q "^${COMMAND} is \\/"; then
```

Comment:
is the intention here to grep for `$COMMAND is \/` or `$COMMAND is /`?

https://github.com/koalaman/shellcheck/wiki/SC1117 implies that it probably should have been `$COMMAND is /`?

Reply:
`$COMMAND is /`, the effect here is the same, it won't be `$COMMAND is \/`

Reply:
Doesn't `\\` put a literal backslash into the text?

Reply:
I thought so but I don't think so now. From SC1117 wiki, looks like it's because we use `"` not `'`.

Reply:
ok, so `'\n'` is correct, and `"\\n"` is the double-quoted equivalent?

Reply:
Seems so!